themes/stanford_basic/src/js/stanford_basic.behavior.js

```diff
@@ -63,6 +63,12 @@ export default {
           .wrap('<li class="su-mobile-site-search"></li>');
       }
 
+      // Check for empty navigation and add space.
+      const $hasNoNav = $('.region-menu', context);
+      if ($hasNoNav.children().length === 0) {
+        $hasNoNav.css( "padding-bottom", "16px" );
```

Comment:
Lets use better variable names and use classes.

```suggestion
      const $menuRegion = $('.region-menu', context);
      if ($menuRegion.children().length === 0) {
        $menuRegion.addClass( "empty-menu" );
```

something like that.